lightninglabs/lnc-web
# Feature/yarn

Switch from `npm` to `yarn` to be more consistent with the other LND projects (`yarn` is used in [lightning-terminal](https://github.com/lightninglabs/lightning-terminal), [builders-guide-sample-app](https://github.com/lightninglabs/builders-guide-sample-app)).

A similar change could also be applied to [lnc-core](https://github.com/lightninglabs/lnc-core).  This project is still using `npm`.

**Test Steps:**

1. To confirm & reproduce the `yarn.lock` file run `yarn import` in each of the directories containing a `package-lock.json` file.
2. Make sure the `yarn` commands and demo applications still work as expected
